front/src/App.vue

```diff
@@ -5,10 +5,25 @@
 import { useGlobalQueryLoading } from '@vue/apollo-composable';
 import { defineComponent, watch } from 'vue';
 
+import SearchDialog from 'src/components/Dialogs/SearchDialog.vue';
+
 export default defineComponent({
   name: 'App',
   mounted() {
     const loading = useGlobalQueryLoading();
+
+    // Maybe we want to move this listener somewhere else
+    // Because this will be rendered/executed when the user is not authenticated as well
+    document.addEventListener('keyup', (e: KeyboardEvent) => {
+      if (!(e.ctrlKey && e.key == 'k')) return;
+
+      e.preventDefault();
+
+      this.$q.dialog({
+        component: SearchDialog,
+      });
+    });
+
```

Comment:
The CTRL+K does not work for me in both Firefox and Chromium. It instead focuses on the address bar to search with my default search engine. I don't know why this does not work. It should be possible since GitHub does the same thing.

Reply:
I now use hotkeys-js to handle hotkeys, it's preferable to handle global keys and os specific ones with this lib